iceoryx_hoofs/test/moduletests/test_objectpool.cpp

```diff
@@ -15,11 +15,9 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-#if !defined(_WIN32)
+#include <gmock/gmock-matchers.h>
```

Comment:
Just small nitpicking. Could you move this include further down so the the objectpool is the first include.

Reply:
actually this header was autoadded by my IDE, I remove it completely.